test/workers.test.js

```diff
@@ -175,7 +175,7 @@ test('tlsOptions using pfx work as intended in workers', { skip: !hasWorkerSuppo
     }
   }, function (err, result) {
     t.error(err)
-    t.ok(result.requests, 'requests are ok')
+    t.ok(result, 'requests are ok')
```

Comment:
does result contain something that suggests that the run succeeded?

Reply:
Yes, if the result is undefined that means it was **not** successful. This particular issue fails with result being undefined, but I've just realised that we also check for existence of an error `t.error(err)` - I think that would be enough.
I'll remove the unnecessary check for the result.

Reply:
you can keep it if that's part of the API that a non-null result means success

Reply:
Okay, then no updates are necessary. That was, indeed, my intention in the first place..